src/main/java/com/github/dadrus/App.java

```diff
@@ -7,10 +7,14 @@
 public class App 
 {
 	// expecting sonar to alert an issue here
-	public static final String PASSWORD = "boo";
+	public static final String OTHER_PASSWORD = "boo";
 	
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
     }
+	
+	public void test() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)